src/components/SetupExtras.vue

```diff
@@ -24,15 +24,15 @@ function handleSubmit() {
 
     <SettingsVotingBlock context="setup" />
 
-    <SettingsValidationBlock context="setup" />
-
     <SettingsDomainBlock context="setup" />
 
     <SettingsSubSpacesBlock context="setup" />
 
     <SettingsTreasuriesBlock context="setup" />
 
     <SettingsPluginsBlock context="setup" />
+
+    <SettingsValidationBlock context="setup" :show-errors="showErrors" />
```

Comment:
What's the reason to move it last?

Reply:
Its hard to find the error if its in the middle 😅 user don't scroll up after clicking "Create" 

While creating a space with non-safe wallet. Proposal validation is at the end, so it is easy to notice 

We have rethink about how we display this error. So i thought for now we can keep this at the end

Reply:
User's screen recording here 
https://discord.com/channels/707079246388133940/1177400622962724924/1177523911089467495

Reply:
Ok yeah for now that works